groovy/gradle.javaee/licenseinfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/gradle/javaee/resources/WebPagesBadge.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/gradle/javaee/resources/javaeeProjectIcon.png</file>
+        <file>src/org/netbeans/modules/gradle/javaee/resources/javaeeProjectIcon_dark.png</file>
+        <license ref="Apache-2.0" />
+        <!-- The complete content of the NOTICE file from: https://raw.githubusercontent.com/gradle/gradle/v6.0.0/subprojects/distributions/src/toplevel/NOTICE -->
+        <notice>
+=========================================================================
+==  NOTICE file corresponding to the section 4 d of                    ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for the Gradle distribution.                          ==
+=========================================================================
+
+This product includes software developed by
+The Apache Software Foundation (http://www.apache.org/).
+
+It includes the following other software:
+
+Groovy (http://groovy-lang.org)
+SLF4J (http://www.slf4j.org)
+JUnit (http://www.junit.org)
+JCIFS (http://jcifs.samba.org)
+
+For licenses, see the LICENSE file.
+
+If any software distributed with Gradle does not have an Apache 2 License, its license is explicitly listed in the
+LICENSE file.
+        </notice>
+        <comment type="COMMENT_UNSUPPORTED">
+            Icons are based based on artwork distributed with Gradle (ALv2 licensed)
+        </comment>
```

Comment:
Can this be expressed more clearly? I failed to find the base graphics for your work (or is it the embedded SVG in the `getting-started.html`)?